packages/platform-ios/src/link/warnAboutPodInstall.ts

```diff
@@ -12,7 +12,7 @@ export default function warnAboutPodInstall(config: Config) {
     .map(depName => {
       const dependency = config.dependencies[depName].platforms.ios;
       return dependency && dependency.podspecPath
-        ? path.basename(dependency.podspecPath).replace(/\.podspec/, '')
+        ? path.basename(dependency.podspecPath || '').replace(/\.podspec/, '')
```

Comment:
This branch is never reached if `dependency.podspecPath` is falsy, because we already guard against it. It was a recent fix, and I'm about to release a patch

Reply:
ah I see! that was fast :)  thx!